dsc/workflows/base/__init__.py

```diff
@@ -93,97 +91,30 @@ def _get_subclasses(cls) -> Iterator[type[Workflow]]:
             yield from subclass._get_subclasses()  # noqa: SLF001
             yield subclass
 
-    def reconcile_bitstreams_and_metadata(self) -> tuple[set[str], set[str]]:
+    @abstractmethod
```

Comment:
I waffled a bit on this, but I've come around to liking the `@abstractmethod` approach here.

Most of the time, we do want workflows to support this, which this nudges towards.  In the event they _don't_, then this docstring pretty clearly outlines what it should do.

Reply:
I agree! Shoutout to @ehanson8 for the great suggestion. 🤓